examples/kotlin-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImInvokeCommand.kt

```diff
@@ -37,17 +37,6 @@ class PairOnNetworkLongImInvokeCommand(
     DiscoveryFilterType.LONG_DISCRIMINATOR
   ) {
   override fun runCommand() {
```

Comment:
we should probably add some comment below that we expect the device to already be paired and that we expect CASE to automatically be established.

Do we still need the `PairingCommand` intermediate then? We seem to pass in a lot of arguments and accept things like setupPinCode and other arguments that PairingComand injects, however if we only need nodeid for CASE we can probably simplify this.

Reply:
Yes, we  still need the PairingCommand intermediately, since cluster subscription is not ready, which still use it.

After we have customized commissioning APIs, then we can depreciate the auto-commissioning API for testing